src/Helpers/LocalMachineHelper.php

```diff
@@ -154,9 +152,13 @@ private function configureProcess(Process $process, $cwd = NULL, $print_output =
    *
    * @return \Symfony\Component\Process\Process
    */
-  protected function executeProcess(Process $process, $callback = NULL, $timeout = NULL): Process {
-    $process->setTimeout($timeout);
-    $process->start(NULL);
+  protected function executeProcess(Process $process, $callback = NULL, $print_output = TRUE): Process {
+    if ($callback === NULL) {
+      $callback = function ($type, $buffer) {
+        $this->output->write($buffer);
```

Comment:
Won't this print output when $print_output is false?

Reply:
Great catch! Our tests should have caught this. I added a test and fixed it to respect `$print_output`.